docs/tools/draco.md

```diff
@@ -0,0 +1,84 @@
+# Draco: a Chemical Data Extractor
```

Comment:
are we OK using this name in publicly facing documentation? i thought this was an internal project name

Reply:
Let ask Max about this